Server/src/Server.Infrastructure/Persistence/QueryHandlers/GetLibrariesWithBookHandler.cs

```diff
@@ -6,11 +6,10 @@
 
 namespace Server.Infrastructure.Persistence.QueryHandlers;
 
-public record GetLibrariesWithBookQuery(Guid Id, int PageSize = 10, int PageNumber = 0, string? OrderByField = null)
-    : IRequest<PaginatedResponseViewModel<LibraryViewModel>>;
+public record GetLibrariesWithBookQuery(Guid Id)
+    : IRequest<List<LibraryViewModel>>;
```

Comment:
tutaj nie zastąpiłeś tych propertisów `PaginationOptions` chyba że tutaj nie paginujemy

Reply:
wydaje mi się że nie ma sensu paginacja tutaj, to używamy w dropdownie do dodawania książki do koszyka